contracts/seed/Seed.sol

```diff
@@ -133,6 +133,19 @@ contract Seed {
         _;
     }
 
+    modifier isClosed() {
+        require(!closed, "Seed: should not be closed");
+        _;
+    }
+
+    modifier isStarted() {
+        require(
+            block.timestamp < startTime,
+            "Seed: class can only be added until startTime"
```

Comment:
```suggestion
    modifier hasNotStarted() {
        require(
            block.timestamp < startTime,
            "Seed: class can only be added until startTime"
```

Either modifier name or require should change.
As I read it, it's the direct opposite.

Reply:
Changed